src/mappings/events/vaults.ts

```diff
@@ -53,8 +53,84 @@ export const vaultsEventKit = (block: any, data: any, module: string, path: stri
     return promises;
   }
 
+  async function updateDailyVaultState(
+    oldState: string | undefined,
+    newState: string,
+    blockTime: Date,
+    blockHeight: number,
+  ): Promise<VaultStatesDaily> {
+    const dateKey = dateToDayKey(blockTime).toString();
+    let vaultState: VaultStatesDaily | undefined = await VaultStatesDaily.get(dateKey);
+
+    if (!vaultState) {
+      const yesterdayDate = new Date(blockTime);
+      yesterdayDate.setDate(yesterdayDate.getDate() - 1);
+      const yesterdayDateKey = dateToDayKey(yesterdayDate).toString();
```

Comment:
What is yesterday's date? Is it the one we have on line 66? or does it become yesterday's date on line 67?

Reply:
it becomes yesterdays date after line 67. unfortunately i dont have a better way of conveying this info